docs/using_the_compiler/README.md

```diff
@@ -312,9 +312,9 @@ The following options can be combined to filter the list of specs to run.
 * `--line <line>`: Runs examples whose line matches `line`.
 * `--location <file>:<line>`: Runs example(s) at `line` in `file` (multiple options allowed).
 * `--tag <tag>`: Runs examples with the specified tag, or excludes examples by adding `~` before the tag (multiple options allowed).
-    * `--tag a --tag b` will include specs tagged with `a` OR `b`.
-    * `--tag ~a --tag ~b` will include specs not tagged with `a` AND not tagged with `b`.
-    * `--tag a --tag ~b` will include specs tagged with `a`, but not tagged with `b`
```

Comment:
And this is a bad change. It turns it into a single list but with unused spacing in the syntax

Reply:
I'm with my linter on this one; with 2 spaces it should be enough to render properly (still I'm undoing it).